lib/Conversion/TorchToLinalg/Linear.cpp

```diff
@@ -641,6 +641,45 @@ class ConvertAtenConvolutionOp : public OpConversionPattern<AtenConvolutionOp> {
 };
 } // namespace
 
+namespace {
+class ConvertAtenAllBoolOp : public OpConversionPattern<AtenAllBoolOp> {
```

Comment:
The lowering for `AtenAnyBoolOp` was recently added, and it is almost the same as the one needed for this op. Can you refactor this code

https://github.com/llvm/torch-mlir/blob/014a6d16c7891d194e9031acba21b85e2a89453e/lib/Conversion/TorchToStd/TorchToStd.cpp#L206-L223

into a helper static function that takes the reduction function as an argument (in your case `llvm::all_of`) so that both lowerings can share as much of the logic as possible?

Reply:
fixed this in the latest commit.